app/serializers/user_serializer.rb

```diff
@@ -33,6 +32,7 @@ def profile
 
   def vet360_contact_information
     person = object.vet360_contact_info
+    return {} if person.blank?
```

Comment:
Return a default if person is nil (which it will be in production until we change the settings flag)